implementation/src/main/java/com/avioconsulting/mule/testing/muleinterfaces/containers/MuleEngineContainer.groovy

```diff
@@ -135,31 +133,11 @@ class MuleEngineContainer {
             log.info 'Using existing services'
             return
         }
-        // In mule 4.1, all you had to do was copy the service JARS to .mule/services
-        // Then mule would unzip them into .mule/.mule/services
-        // In 4.2.0, that doesn't work any more (you'll see scheduler service complaints)
-        // we have to unzip the JARs
-        if (muleVersion.startsWith('4.1')) {
-            copyServicesFor41(services,
-                              servicesDir)
-        } else {
-            unzipServicesFor420(services,
-                                servicesDir)
-        }
+        unzipServicesFor420(services,
```

Comment:
Minor thing, since we've eliminated previous version support should this just be unzipServices ?

Reply:
I was not sure how much of a breadcrumb to leave here in case this has to be done again for future versions